src/test/java/com/google/cloud/tools/appengine/cloudsdk/serialization/CloudSdkVersionTest.java

```diff
@@ -41,9 +44,18 @@ public void testConstructor_emptyString() {
     new CloudSdkVersion("");
   }
 
-  @Test(expected = NumberFormatException.class)
-  public void testConstructor_nonNumeric() {
-    new CloudSdkVersion("v1beta3-1.0.0");
+  @Test
+  public void testConstructor_invalid() {
```

Comment:
This is going to feel academic, but generally with unit tests, each test case should be as specific as possible. Is there something qualitatively different about the inputs provided?  Are they different edges?  If so, put them in different tests and label them clearly.